dr/discovered/deployment-rbd/placement.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  annotations:
+    cluster.open-cluster-management.io/experimental-scheduling-disable: "true"
+  name: placement
+spec:
+  clusterSets:
+    - default
+  numberOfClusters: 1
+  predicates:
+    - requiredClusterSelector:
+        claimSelector:
+          matchExpressions:
+            - key: name
+              operator: In
+              values:
+                - dr1
```

Comment:
predicates removed since ocm is not handling this placement and ramen does not need it.